File: Code/Classic Games Server/Server/src/Server/Parchis/Objects/Player.java
package Server.Parchis.Objects;

public class Player {

	private TeamColor color;
	
	private ParchisUser user;
	
	public Player(TeamColor color, ParchisUser user) {
		this.color = color;
		this.user = user;
	}
	
	public ParchisUser getUser() {
		return user;
	}
	
	public String getName() {
		return user.getData().getName();
	}
	
	public TeamColor getColor() {
		return color;
	}
	
	@Override
	public String toString() {
		return color.name();
	}
}

File: Code/Classic Games/Assets/Scripts/Server/Parchis/Objects/Player.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

namespace PARCHIS {
    public class Player : MonoBehaviour {
        private string playerName;
        private TeamColor color;

        public void initPlayer(string playerName, TeamColor color) {
            this.playerName = playerName;
            this.color = color;
        }

        public TeamColor getColor() {
            return color;
        }

        public string getPlayerName() {
            return playerName;
        }
    }
}
